python/timeplus/version.py

```diff
@@ -1 +1 @@
-__version__ = "2.0.0b1"   # noqa: F401
+__version__ = "2.0.0b3"   # noqa: F401
```

Comment:
2.0.0b3 has released yesterday, do you want to set this as b4?

Reply:
this version will be overriden when run build automatically